lite/examples/style_transfer/android/app/download_model.gradle

```diff
@@ -4,24 +4,38 @@ task downloadPredictModelFile(type: Download) {
     overwrite false
 }
 
-task downloadTransformModelDynamicFile(type: Download) {
-    src 'https://storage.googleapis.com/download.tensorflow.org/models/tflite/arbitrary_style_transfer/style_transfer_quantized_dynamic.tflite'
-    dest project.ext.ASSET_DIR + '/style_transfer_quantized_dynamic.tflite'
-    overwrite false
-}
+// TODO: Add support for dynamic size model
+//task downloadTransformModelDynamicFile(type: Download) {
```

Comment:
If it's not being used for now, can we delete this block and add it back later, instead of commenting out?

Reply:
Done.